share/translations/measurements_nl_NL.ts

```diff
@@ -2437,43 +2437,43 @@
         <location filename="../../src/libs/vpatterndb/vtranslatemeasurements.cpp" line="1278"/>
         <source>From Shoulder Tip to Shoulder Tip, across front.</source>
         <comment>Full measurement description.</comment>
-        <translation>Van schouderpunt naar schouderpunt, kruislings voorkant.</translation>
+        <translation>Van schouderpunt naar schouderpunt, langs voorkant.</translation>
     </message>
     <message>
         <location filename="../../src/libs/vpatterndb/vtranslatemeasurements.cpp" line="1282"/>
         <source>across_chest_f</source>
         <comment>Name in a formula. Don&apos;t use math symbols and space in name!!!!</comment>
-        <translation>Gekruist_borst_voorkant</translation>
+        <translation>Breedte_borst_voorkant</translation>
```

Comment:
all other formula names start with lowercase, maybe adapt this as well (while at it)